src/proxy.js

```diff
@@ -26,12 +26,53 @@ module.exports = function proxyPolyfill() {
     return o ? (typeof o === 'object' || typeof o === 'function') : false;
   }
 
+  const $Object = Object;
```

Comment:
Why do we need to shadow `Object` here?

Reply:
This is necessary, since otherwise **Closure** will make `!({ __proto__: null } instanceof Object)` into a constant condition, that evaluates to `true`, since **Closure** assumes that all environments support [`__proto__` Property Names in Object Initializers](https://tc39.es/ecma262/#sec-__proto__-property-names-in-object-initializers)